app/src/main/java/org/fossasia/susi/ai/signup/SignUpPresenter.kt

```diff
@@ -137,14 +137,6 @@ class SignUpPresenter(signUpActivity: SignUpActivity) : ISignUpPresenter, ISignU
         signUpView = null
     }
 
-    override fun loginLogout() {
```

Comment:
This will remain here but delegated to the new presenter

Reply:
@batbrain7 @arundhati24 @iamareebjamal what is the need of keeping this here when we have a separate presenter file that handles all the login operations? Isn't it just writing just extra lines of codes?

Reply:
Read about delegation

Reply:
And no, no lines will be repeated

Reply:
I will surely look into it sir